kinetic/distribution.yaml

```diff
@@ -3038,7 +3038,7 @@ repositories:
       tags:
         release: release/kinetic/{package}/{version}
       url: https://github.com/UTNuclearRoboticsPublic/move_base_to_manip-release.git
-      version: 1.0.6-2
+      version: 1.0.9-0
```

Comment:
The release repo does not contain a release with that version number for Kinetic. Please make sure that you have run `bloom`.